src/panels/lovelace/editor/config-elements/hui-weather-forecast-card-editor.ts

```diff
@@ -0,0 +1,100 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+
+import { struct } from "../../common/structs/struct";
+import { EntitiesEditorEvent, EditorTarget } from "../types";
+import { hassLocalizeLitMixin } from "../../../../mixins/lit-localize-mixin";
+import { HomeAssistant } from "../../../../types";
+import { LovelaceCardEditor } from "../../types";
+import { fireEvent } from "../../../../common/dom/fire_event";
+import { Config } from "../../cards/hui-alarm-panel-card";
+import { configElementStyle } from "./config-elements-style";
+
+import "../../../../components/entity/ha-entity-picker";
+
+const cardConfigStruct = struct({
+  type: "string",
+  entity: "string?",
```

Comment:
Should be a weather entity, not just a string. And entity should be "entity-id" https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/panels/lovelace/common/structs/struct.ts#L7

Reply:
And I guess it is not optional?

Reply:
I set it to optional as it would error out to the yaml editor otherwise, but setting a default should fix that